CHANGELOG.md

```diff
@@ -15,6 +15,12 @@ As a result of the changes for **CUMULUS-1193** and **CUMULUS-1264**, **you must
 
 ### BREAKING CHANGES
 
+- **CUMULUS-1228**
+  - The default AMI used by ECS instances is now an NGAP-compliant AMI. This
```

Comment:
So the default AMI, is the current (as in right now) NGAP AMI. Do we have documentation on how to find the AMI Id of the latest NGAP AMI?

I think in addition to having this info here, it should go in the deployment docs on this page: https://nasa.github.io/cumulus/docs/deployment/config_descriptions

Reply:
Updated in d9eebbb57